src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/github/v4/GraphqlCheckRunProvider.java

```diff
@@ -222,6 +231,42 @@ private void postSummaryComment(String apiUrl, Map<String, String> headers, Stri
 
     }
 
+    private void minimizeComment(String graphqlUrl, Map<String, String> headers, String commentId) {
+
+        try {
```

Comment:
Nitpick: the IOException can only happen on the `executeRequest` as far as I can tell, so keep the try block constrained to that line rather than covering the whole method.